### PR TITLE
Add unique check to motion_comment.create

### PR DESCRIPTION
**Pull request by reiterl**
Add a test.

### CODE REVIEW

File: openslides_backend/action/actions/motion_comment/create_delete_update.py [hunk 1]
```diff
@@ -55,6 +56,19 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
...
+        instance = super().update_instance(instance)
+        # check, if (user_id, motion_id) already in the datastore.
+        filter_ = And(
+            FilterOperator("section_id", "=", instance["section_id"]),
+            FilterOperator("motion_id", "=", instance["motion_id"]),
+            FilterOperator("meeting_id", "=", instance["meeting_id"]),
+        )
+        exists = self.datastore.exists(collection=self.model.collection, filter=filter_)
+        if exists:
+            raise ActionException("(section_id, motion_id) must be unique.")
```
Comment (jsangmeister) [inline openslides_backend/action/actions/motion_comment/create_delete_update.py:69]:
```suggestion
            raise ActionException("There already exists a comment for this section, please update it instead.")
```

File: openslides_backend/action/actions/motion_comment/create_delete_update.py [hunk 2]
```diff
@@ -55,6 +56,19 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
 class MotionCommentCreate(PermissionMixin, CreateActionWithInferredMeeting):
     relation_field_for_meeting = "motion_id"
 
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().update_instance(instance)
+        # check, if (user_id, motion_id) already in the datastore.
```
Comment (jsangmeister) [inline openslides_backend/action/actions/motion_comment/create_delete_update.py:61]:
```suggestion
        # check, if (section_id, motion_id) already in the datastore.
```

**Review by jsangmeister — CHANGES_REQUESTED** [review]
minor changes

File: openslides_backend/action/actions/motion_comment/create_delete_update.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion_comment/create_delete_update.py:69]: Done.

File: openslides_backend/action/actions/motion_comment/create_delete_update.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion_comment/create_delete_update.py:61]: Done.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**